src/sdk/PnP.Core.Auth.Test/Providers/OnBehalfOfAuthenticationProviderTests.cs

```diff
@@ -391,6 +434,30 @@ private static OnBehalfOfAuthenticationProvider PrepareOnBehalfOfAuthenticationP
             return provider;
         }
 
+        private static OnBehalfOfAuthenticationProvider PrepareOnBehalfOfAuthenticationProviderWithCertificateFile()
+        {
+            if (TestCommon.RunningInGitHubWorkflow())
```

Comment:
@jansenbe I am not sure if it is possible to load the cert from test assets within a GitHub action